MoviesTVSentiments/app/src/main/java/com/google/moviestvsentiments/model/Asset.java

```diff
@@ -0,0 +1,52 @@
+package com.google.moviestvsentiments.model;
+
+import androidx.annotation.NonNull;
+import androidx.room.ColumnInfo;
+import androidx.room.Entity;
+import androidx.room.TypeConverters;
+
+/**
+ * A record in the assets database table.
+ */
+@Entity(tableName = "assets_table", primaryKeys = {"asset_id", "asset_type"})
+@TypeConverters({AssetType.class})
+public class Asset {
```

Comment:
The fields below differ a bit from the Room DB table described in your design doc. banner and expires are missing. Do you plan on adding them later?

Reply:
`expires` should not be there. I will add all the extra fields as described in the `Room database` section of the design doc when I handle data syncing with the server. `banner` was accidentally left off.

Reply:
Make sense.